docs/core/diagnostics/dotnet-stack.md

```diff
@@ -0,0 +1,116 @@
+---
+title: dotnet-stack tool - .NET Core
+description: Installing and using the dotnet-stack command-line tool.
+ms.date: 11/21/2019
+---
+# dotnet-stack performance analysis utility
+
+**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
```

Comment:
Wasn't there a runtime bug in 3.0 that prevented this from working so this only works from 3.1, or am I misremembering this?

Reply:
That was gcdump!

Reply:
you're right :) nvm then!